.travis.yml

```diff
@@ -23,8 +23,7 @@ android:
     - addon-google_apis-google-19
 
     #System images
-    - sys-img-armeabi-v7a-android-22
-    - sys-img-armeabi-v7a-android-17
+    - sys-img-armeabi-v7a-android-16
```

Comment:
This should speed up the build somewhat, version 16 is supposedly the fastest image.